packages/salesforcedx-vscode-core/src/telemetry/telemetry.ts

```diff
@@ -164,6 +164,16 @@ export class TelemetryService {
     }
   }
 
+  public sendEventData(
```

Comment:
Need to add a test for this in `test/vscode-integration/telemetry/index.test.ts` since no other method is testing sending measures.